crates/sui-core/src/unit_tests/gas_tests.rs

```diff
@@ -486,36 +460,6 @@ async fn test_move_call_gas() -> SuiResult {
     assert!(gas_cost.storage_cost > 0 && gas_cost.storage_cost < gas_cost.storage_rebate);
     // Check that we have storage rebate that's the same as previous cost.
     assert_eq!(gas_cost.storage_rebate, prev_storage_cost);
-    let expected_gas_balance = expected_gas_balance - gas_cost.gas_used() + gas_cost.storage_rebate;
```

Comment:
this was not doing what the comment was saying and removing the min charge makes those tests hard.
We should have them but not how this is